environment.yml

```diff
@@ -35,8 +35,11 @@ dependencies:
   - nbconvert>=5.4.1
   - nbsphinx
   - pandoc
-  - dask
-  - distributed
+  - dask-core
+  - toolz>=0.7.3
+  - fsspec>=0.5.1
+  - partd>=0.3.10
+  - cloudpickle>=0.2.1
```

Comment:
Can you add a comment here that those last 4 ones are just dependencies of dask.dataframe?